### ISSUE TITLE
StdConfig setCmdLineArgs behaves as 'add' not 'set'

### ISSUE THREAD

**Issue by eeverman** [eeverman/andhow#509]
**Describe the bug**
Calling setCmdLineArgs will only add cmdLine Args, it does not replace the existing ones.
Based on the name and documentation, this is incorrect - it should remove existing args.

Also, the unit test (StdConfigTest) should be updated to check this condition.

**Comment by alex-kar** [comment]
@eeverman What's the expected behavior for empty and null arguments? I assume if arguments are empty or null it should remove old ones and getCmdLineArgs() should return empty list. Is it correct? In current implementation null argument just ignored.

**Comment by eeverman** [comment]
@alex-kar Yes, you are right.  I updated the issue description to reflect that.  I'll assign this one to you - let me know if you didn't want it.
